### PR TITLE
feat: Landscape profile

### LINKED ISSUE #206: Landscape Profile: Taxonomies
## Description
- Create taxonomy values table
- Add taxonomy values relationship table with other entity types

### LINKED ISSUE #207: Landscape Profile: Relationships with groups
## Description
We should be able to define relationships between groups and landscapes that show if a group is a partnership or if a group is affiliated with a landscape

### LINKED ISSUE #209: Landscape Profile:  Profile fields
## Description
- Add all profile fields. See: https://www.figma.com/file/HsAlxOnHkABJ3IyT2D0SRW/Create%2FUpdate-Landscape-Profile_V2-R1.2

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Added support for new landscape fields
- Taxonomy terms
- Partnership relationship

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added

### Related Issues
Fixes #209 
Fixes #207
Fixes #206

### CODE REVIEW

File: terraso_backend/apps/core/models/landscapes.py [hunk 1]
```diff
@@ -40,6 +43,26 @@ class Landscape(SlugModel):
...
+
+    PARTNERSHIP_STATUS_NO = "no"
+    PARTNERSHIP_STATUS_IN_PROGRESS = "in-progress"
+    PARTNERSHIP_STATUS_YES = "yes"
+    DEFAULT_PARTNERSHIP_STATUS = PARTNERSHIP_STATUS_NO
+
+    MEMBERSHIP_TYPES = (
+        (PARTNERSHIP_STATUS_NO, _("No")),
+        (PARTNERSHIP_STATUS_IN_PROGRESS, _("In Progress")),
+        (PARTNERSHIP_STATUS_YES, _("Yes")),
```
Comment (paulschreiber) [inline terraso_backend/apps/core/models/landscapes.py:58]: question: do we need to use `_()` here? Are we doing any localization in the backend?

File: terraso_backend/apps/core/models/taxonomy_terms.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    TYPE_LIVELIHOOD = "livelihood"
+    TYPE_COMMODITY = "commodity"
+    TYPE_ORGANIZATION = "organization"
+
+    TYPES = (
+        (TYPE_ECOSYSTEM_TYPE, _("Ecosystem Type")),
+        (TYPE_LANGUAGE, _("Language")),
+        (TYPE_LIVELIHOOD, _("Livelihood")),
+        (TYPE_COMMODITY, _("Commodity")),
+        (TYPE_ORGANIZATION, _("Organization")),
```
Comment (paulschreiber) [inline terraso_backend/apps/core/models/taxonomy_terms.py:20]: same as above.

**Review by paulschreiber — APPROVED** [review]

File: terraso_backend/apps/core/models/landscapes.py (hunk 1 above)
Comment (josebui) [reply terraso_backend/apps/core/models/landscapes.py:58]: @paulschreiber This is how the other fields with choices are handled, I think this is just used for the admin panel right now

File: terraso_backend/apps/core/models/taxonomy_terms.py (hunk 2 above)
Comment (josebui) [reply terraso_backend/apps/core/models/taxonomy_terms.py:20]: check the other message answer

**Pull request merged**